src/components/dialogs/parameters-confirm-closing-dialog.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import PropTypes from 'prop-types';
+
+import {
+    Button,
+    Dialog,
+    DialogContent,
+    DialogContentText,
+    DialogActions,
+} from '@mui/material';
+
+function ParametersConfirmClosingDialog({ open, onClickYes, closeDialog }) {
+    const handleClickYes = () => {
+        onClickYes();
+        closeDialog();
+    };
+
+    return (
+        <Dialog open={open} onClose={() => {}}>
```

Comment:
Maybe you wanna call closeDialog for onClose dialog props

```
<Dialog open={open} onClose={closeDialog}>
```

Reply:
No, in this particular Dialog, I want the user to explicitly select an answer to close the Dialog.